src/portable/create.rs

```diff
@@ -149,12 +150,13 @@ pub fn create(cmd: &Create, opts: &crate::options::Options) -> anyhow::Result<()
             || anyhow::Ok(Query::stable()),
         )?;
         let inst = install::version(&query).context("error installing EdgeDB")?;
+        let specific_version = &inst.version.specific();
         let info = InstanceInfo {
             name: name.clone(),
             installation: Some(inst),
             port,
         };
-        bootstrap(&paths, &info, &cmd.default_user)?;
+        bootstrap(&paths, &info, &cmd.default_user, cmd.default_branch.as_ref().unwrap_or(&get_default_branch_name(specific_version)))?;
```

Comment:
This will call `get_default_branch_name()` even if `cmd.default_branch` is `Some`.

Use `.unwrap_or_else()` instead.